src/components/LiquidfillChart/plugView.js

```
@@ -1,537 +1,358 @@
+          "amplitude",
+          "cx",
+          "cy",
+          "phase",
+          "radius",
+          "radiusY",
+          "waterLevel",
+          "waveLength",
+        ];
+        for (var l = 0; l < s.length; ++l) {
+          var n = s[l];
+          if (r.shape.hasOwnProperty(n)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)